app/controllers/ApplicationController.scala

```diff
@@ -27,14 +27,14 @@ class ApplicationController @Inject() (
    *
    * @return The result to display.
    */
-  def user = SecuredAction.async { implicit request =>
+    def user : Action[AnyContent] = SecuredAction.async { implicit request =>
```

Comment:
Can we use "Action[JsValue]" here like you used in other actions, to make things consistent? All return the same type of response.